csharp/downgrades/66044cfa5bbf2ecfabd06ead25e91db2bdd79764/expressions.ql

```diff
@@ -0,0 +1,13 @@
+class Expression extends @expr {
+  string toString() { none() }
+}
+
+class TypeOrRef extends @type_or_ref {
+  string toString() { none() }
+}
+
+from Expression e, int k, int kind, TypeOrRef t
+where
+  expressions(e, k, t) and
+  if k = 138 then kind = 106 else kind = k
+select e, kind, t
```

Comment:
Should this select the insert part of `e` instead of `e`?

Reply:
I considered doing that as well, but it requires that we also make changes to the `expr_parent` relation as well (this is what I am vaguely hinting in the PR description).
Basically, I am worried that I break some DB invariant by changing the parent/child relationship as a part of the script.
Should we attempt to make the better downgrade script? It requires
(1) Removing the *insert* expressions from the expressions relation
(2) Adjust the `expr_parent` relation
(3) Consider if we need to do something with the new dangling *align* and *format* expressions.